src/support/support.service.js

```diff
@@ -1,6 +1,9 @@
 angular.module("ovh-api-services").service("OvhApiSupport", function ($injector) {
     "use strict";
     return {
+        iceberg: function () {
```

Comment:
In order to make consistency:

```suggestion
        Iceberg: function () {
```

Reply:
you're 100% right !